### ISSUE TITLE
[Enhancement]: Can we change the milvus base image to ubuntu20?

### ISSUE THREAD

**Issue by shivabohemian** [milvus-io/milvus#36991]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to be added?

Can we change the base image of Milvus from Ubuntu 22 to Ubuntu 20? 

### Why is this needed?

The `glibc` version in Ubuntu 22 is too high for some users who use the binary packages directly.

**Comment by xiaofan-luan** [comment]
could ubuntu 20 + some older version of glibc works?

**Comment by shivabohemian** [comment]
Of course infinitely forward compatibility is not appropriate.
I found that the base image of milvus for many previous versions was ubuntu20, and its version of glibc is available for ubuntu20+ and debian11+, which is probably the most popular linux distribution today.

**Comment by shivabohemian** [comment]
I am currently deploying Milvus using binary packages. According to this documentation ([README.md](https://github.com/milvus-io/milvus/blob/master/deployments/binary/README.md)), I couldn't find the officially uploaded Milvus binary package, so I extracted it from the image. In Debian 11, the glibc version is 2.31, the same as in Ubuntu 20, while in Ubuntu 22 it is 2.35. Based on the above, I can only choose to compile Milvus myself, but recompiling is rather cumbersome.

I noticed that the project offers multiple build environments, including Ubuntu 20 and Ubuntu 22. Based on my observations of download repositories for projects like MySQL, they provide packages for the highest Linux-generic version with glibc 2.28, as well as versions with glibc 2.17 and 2.12. I believe this brings great convenience to users. It would be fantastic if the Milvus community could provide precompiled binary packages for different glibc versions.

Of course, this is just my suggestion, and I really appreciate it.
